server/api/crud/model_monitoring/deployment.py

```diff
@@ -1000,6 +1008,11 @@ def _delete_model_monitoring_stream_resources(
                     topics=topics,
                     error=mlrun.errors.err_to_str(e),
                 )
+            except kafka.errors.NoBrokersAvailable as e:
+                # Raise an error that will be caught by the called and skip the deletion of the stream
```

Comment:
If it will always be caught by the caller, why even bother raising it and catching it again? Also, a typo:
```suggestion
                # Raise an error that will be caught by the caller and skip the deletion of the stream
```